landing-page/content/common/spark-quickstart.md

```diff
@@ -0,0 +1,370 @@
+schema = StructType([
+  StructField("VendorID", LongType(), True),
+  StructField("TripID", LongType(), True),
+  StructField("Trip_distance", FloatType(), True),
+  StructField("Fare_amount', DoubleType(), True),
+  StructField("Store_and_fwd_flag', StringType(), True)
+])
+
+df = spark.createDataFrame([], schema)
+df.writeTo("prod.nyc.taxis").create()
+```
+{{% /tabcontent %}}
+{{% /codetabs %}}
+
+Iceberg catalogs support the full range of SQL DDL commands, including:
+
+* [`CREATE TABLE ... PARTITIONED BY`](../spark-ddl#create-table)
+* [`CREATE TABLE ... AS SELECT`](../spark-ddl#create-table--as-select)
+* [`ALTER TABLE`](../spark-ddl#alter-table)
+* [`DROP TABLE`](../spark-ddl#drop-table)
+
+### Writing
+
+Once your table is created, insert data using [`INSERT INTO`](../spark-writes#insert-into):
+
+{{% codetabs "InsertData" %}}
+{{% addtab "SparkSQL" checked %}}
+{{% addtab "SparkShell" %}}
+{{% addtab "PySpark" %}}
+{{% tabcontent "SparkSQL"  %}}
+```sql
+INSERT INTO prod.nyc.taxis
+VALUES (1, 1000371, 1.8, 15.32, 'N'), (2, 1000372, 2.5, 22.15, 'N'), (2, 1000373, 0.9, 9.01, 'N'), (1, 1000374, 8.4, 42.13, 'Y');
+```
+{{% /tabcontent %}}
+{{% tabcontent "SparkShell" %}}
+```scala
+import org.apache.spark.sql.Row
+
+val schema = spark.table("prod.nyc.taxis").schema
+val data = Seq(
+    Row(1: Long, 1000371: Long, 1.8f: Float, 15.32: Double, "N": String),
+    Row(2: Long, 1000372: Long, 2.5f: Float, 22.15: Double, "N": String),
+    Row(2: Long, 1000373: Long, 0.9f: Float, 9.01: Double, "N": String),
+    Row(1: Long, 1000374: Long, 8.4f: Float, 42.13: Double, "Y": String)
+)
+val df = spark.createDataFrame(spark.sparkContext.parallelize(data), schema)
+df.writeTo("prod.nyc.taxis").append()
+```
+{{% /tabcontent %}}
+{{% tabcontent "PySpark" %}}
+```py
+schema = spark.table("prod.nyc.taxis").schema
+data = [
+    (1, 1000371, 1.8, 15.32, "N"),
+    (2, 1000372, 2.5, 22.15, "N"),
+    (2, 1000373, 0.9, 9.01, "N"),
+    (1, 1000374, 8.4, 42.13, "Y")
+  ]
+df = spark.createDataFrame(data, schema)
+df.writeTo("prod.nyc.taxis").append()
+```
+{{% /tabcontent %}}
+{{% /codetabs %}}
+
+### Reading
+
+To read a table, simply use the Iceberg table's name:
+
+{{% codetabs "SelectData" %}}
+{{% addtab "SparkSQL" checked %}}
+{{% addtab "SparkShell" %}}
+{{% addtab "PySpark" %}}
+{{% tabcontent "SparkSQL"  %}}
+```sql
+SELECT * FROM prod.nyc.taxis;
+```
+{{% /tabcontent %}}
+{{% tabcontent "SparkShell" %}}
+```scala
+val df = spark.table("prod.nyc.taxis").show()
+```
+{{% /tabcontent %}}
+{{% tabcontent "PySpark" %}}
+```py
+df = spark.table("prod.nyc.taxis").show()
+```
+{{% /tabcontent %}}
+{{% /codetabs %}}
+
+For [inspecting tables](../spark-queries#inspecting-tables) SQL is recommended. To view all of the snapshots in a table, use the `snapshots` metadata table:
+```sql
+SELECT committed_at, snapshot_id, parent_id, operation FROM prod.nyc.taxis.snapshots;
+```
+```
```

Comment:
we could use a table markdown to display the results

Reply:
Oh this looks much better, thank you!